_data/conferences.yml

```diff
@@ -23,6 +23,36 @@
 #
 ##########################
 
+- name: adiOS
+  link: tbd
+  start: 2022-12-18
+  end: 2020-12-18
+  location: 🇰🇷 Seoul, South Korea
+  cocoa-only: true
+
+- name: Let’Swift
+  link: http://letswift.kr/2022/
```

Comment:
Is there an HTTPS version of the website?

Reply:
I searched it and they have! I will update it